classes/LwcSalagaController.cls

```diff
@@ -0,0 +1,136 @@
+public with sharing class LwcSalagaController {
+	public static final String FIRST_NAME_API_NAME = 'FirstName';
+	public static final String LAST_NAME_API_NAME = 'LastName';
+	public static final String LANGUAGE_API_NAME = 'Language';
+
+	public static final String FIRST_NAME_LABEL = 'First Name';
+	public static final String LAST_NAME_LABEL = 'Last Name';
+	public static final String LANGUAGE_LABEL = 'Language';
+
+	public static final String LAST_MODIFIED_BY_LABEL = 'Last Modified By';
+
+	public class UserInfoField {
+		@AuraEnabled
+		public String apiName;
+		@AuraEnabled
+		public String value;
+		@AuraEnabled
+		public String label;
+
+
+		public UserInfoField(String apiName, String value, String label) {
+			this.apiName = apiName;
+			this.value = value;
+			this.label = label;
+		}
+	}
+
+	public class ObjectInfo {
+		@AuraEnabled
+		public String objectApiName;
+		@AuraEnabled
+		public String objectLabel;
+
+		public ObjectInfo(String objectApiName, String objectLabel) {
+			this.objectApiName = objectApiName;
+			this.objectLabel = objectLabel;
+		}
+	}
+
+	public class ObjectWrapper {
+		@AuraEnabled
+		public String objectApiName;
+		@AuraEnabled
+		public List<FieldWrapper> records;
+
+		public ObjectWrapper(String objectApiName, List<FieldWrapper> recordIds) {
+			this.objectApiName = objectApiName;
+
+			this.records = recordIds;
+		}
+	}
+
+	public class FieldWrapper {
+		@AuraEnabled
+		public String Id;
+		@AuraEnabled
+		public String name;
+		@AuraEnabled
+		public String lastModifiedById;
+
+		public FieldWrapper(String Id, String name, String lastModifiedById) {
+			this.Id = Id;
+			this.name = name;
+			this.lastModifiedById = lastModifiedById;
+		}
+	}
+
+	@AuraEnabled(Cacheable=true)
+	public static List<UserInfoField> getUserInfoFields() {
+		List<UserInfoField> infoFields = new List<UserInfoField>();
+
+		UserInfoField firstName = new UserInfoField(FIRST_NAME_API_NAME, UserInfo.getFirstName(), FIRST_NAME_LABEL);

```

Comment:
instead of use the same lines of the same code, you can use List/Set of fields of User and for loop and initialize them.


`
Set<String> setFields = new Set<String> {'field1', 'field2', 'field3'};
for (String field: setFields) {
infoFields.add(new UserInfoField(field, UserInfo.getFieldName(), describe.getLabel(field));
}
`